code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -294,12 +294,30 @@ obj/item/asteroid/basilisk_hide/New()
 	icon = 'icons/obj/food.dmi'
 	icon_state = "boiledrorocore"
 	var/inert = 0
+	var/timer = 60
 
 /obj/item/asteroid/hivelord_core/New()
 	..()
-	spawn(1200)
+	create_reagents(5)
+	processing_objects.Add(src)
+
+/obj/item/asteroid/hivelord_core/process()
+	if(reagents && reagents.has_reagent(FROSTOIL, 5))
```

Comment:
He edited syringes.dm to include the hivelord core in its `var/injectable_types` list. It's the same thing that lets you inject lamarr with creatine and what not
Should probably be a proc on the object itself though or something